Assets/HoloToolkit-Examples/Prototyping/Shaders/UnlitTransparentTriplanar.shader

```diff
@@ -52,9 +54,10 @@ Shader "MixedRealityToolkit/Examples/UnlitTransparentTriplanar"
 			fixed3 _LightDir;
 			float1 _LightIntensity;
 			float1 _CameraIntensity;
-			UNITY_INSTANCING_CBUFFER_START(Props)
+			UNITY_INSTANCING_BUFFER_START(Props)
```

Comment:
These changes should be reverted to prevent backwards capability breaking for projects that target 5.6

Reply:
Maybe I done some changes accidentally, this part of coding I made no changes too.